### PR TITLE
Document `/search/project/id` endpoint with OpenAPI

### PR DESCRIPTION
**Pull request by eduardoj**

### CODE REVIEW

File: src/api/public/apidocs-new/OBS-v2.10.50.yaml [hunk 1]
```diff
@@ -175,6 +175,10 @@ paths:
     $ref: 'paths/search_missing_owner.yaml'
   /search/owner:
     $ref: 'paths/search_owner.yaml'
+  /search/package/id:
+    $ref: 'paths/search_package_id.yaml'
```
Comment (dmarcoux) [inline src/api/public/apidocs-new/OBS-v2.10.50.yaml:179]: This file is missing, no?

Comment (eduardoj) [reply src/api/public/apidocs-new/OBS-v2.10.50.yaml]: That was a leftover of a previous and more aggressive approach to this PR. :sweat_smile: Removed.

File: src/api/public/apidocs-new/components/parameters/search_limit.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+in: query
+name: limit
+schema:
+  type: integer
+description: TODO
```
Comment (dmarcoux) [inline src/api/public/apidocs-new/components/parameters/search_limit.yaml:5]: To do or not to do... this is the question! :smile:

Same thing for the file *src/api/public/apidocs-new/components/parameters/search_offset.yaml*. Both are missing a description.

Comment (eduardoj) [reply src/api/public/apidocs-new/components/parameters/search_limit.yaml]: Ops, true. Fixed.

**Review by dmarcoux — APPROVED** [review]

**Pull request merged**